### PR TITLE
add atteniton_rnn.ipynb and some functions in .py files

### PR DESCRIPTION
**Pull request by oh-yu**

### CODE REVIEW

**Comment by oh-yu** [comment]
- model.py
- [x] what is p??
- [x] after 「attention = (h_t*hs).sum(axis=2)」, we need softmax

- train_model.py
- [x] no need comentout(#torch.nn.utils.clip_grad_norm(model.parameters(), max_norm=1))
